app/src/main/java/org/torproject/android/OrbotMainActivity.java

```diff
@@ -885,7 +889,6 @@ else if (request == REQUEST_VPN && response == RESULT_OK) {
         }
         else if (request == REQUEST_VPN && response == RESULT_CANCELED) {
             mBtnVPN.setChecked(false);
-            Prefs.putUseVpn(false);
```

Comment:
Got rid of this line because the event listener on `mBtnVPN` makes this preference setting call